wiz-admission-controller/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for wiz-admission-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+wizApiToken:
+  clientId: ""
+  clientToken: ""
+  env: "prod" # (prod/ gov)
+
+replicaCount: 2
+
+image:
+  repository: docker.io/library/wiz-admission-controller-server
```

Comment:
I'd be more convenient to separate the registry (`docker.io/library`)  and the repository (`wiz-admission-controller-server`) into different overrideable values.